src/calculateRentalCost.js

```diff
@@ -4,7 +4,29 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const ONE_DAY_COST = 40;
+  const LONG_TERM = 7;
+  const SHORT_TERM = 3;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  let totalAmount = 0;
+
+  if (days >= LONG_TERM) {
+    totalAmount = ONE_DAY_COST * days - LONG_TERM_DISCOUNT;
+
+    return totalAmount;
+  }
+
+  if (days >= SHORT_TERM) {
+    totalAmount = ONE_DAY_COST * days - SHORT_TERM_DISCOUNT;
+
+    return totalAmount;
```

Comment:
u can use this option
```suggestion
    return DEFAULT_PRICE - SHORT_TERM_DISCOUNT;
```